RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -2214,6 +2219,19 @@ boolean isGeneralStoreAvailable()
 	return true;
 }
 
+boolean isMusGuildStoreAvailable()
+{
+	if(($classes[seal clubber, turtle tamer] contains my_class()) && guild_store_available())
+	{
+		return true;
+	}
+	if ((my_class() == $class[accordion thief]) && (my_level() >= 9) && guild_store_available())
```

Comment:
Just an FYI, you don't need all those brackets.
`if ($classes[seal clubber, turtle tamer] contains my_class() && guild_store_available())`
and
`if (my_class() == $class[accordion thief] && my_level() >= 9 && guild_store_available())`
are the same as your code and should be more easily readable because you don't have to parse where one logic chunk starts and the next ends.
Only use extra brackets when you have discrete chunks of logic you need to separate e.g.
`if (this == that && (something() || value < max))`

Reply:
change made. 

in regards to ease of readability, personally I find it more easily readable the way I wrote it. but if the preferred notation is to not include it then I won't